quiche/src/recovery/mod.rs

```diff
@@ -318,39 +74,22 @@ impl LossDetectionTimer {
         self.time = None;
     }
 }
-pub struct Recovery {
-    epochs: [RecoveryEpoch; packet::Epoch::count()],
-
-    loss_timer: LossDetectionTimer,
-
-    pto_count: u32,
-
-    rtt_stats: RttStats,
-
-    pub lost_spurious_count: usize,
-
-    pkt_thresh: u64,
-
-    time_thresh: f64,
-
-    bytes_in_flight: usize,
-
-    bytes_sent: usize,
-
-    pub bytes_lost: u64,
-
-    max_datagram_size: usize,
-
-    #[cfg(feature = "qlog")]
-    qlog_metrics: QlogMetrics,
-
-    /// How many non-ack-eliciting packets have been sent.
-    outstanding_non_ack_eliciting: usize,
-
-    congestion: Congestion,
 
-    /// A resusable list of acks.
-    newly_acked: Vec<Acked>,
+impl std::fmt::Debug for LossDetectionTimer {
+    fn fmt(&self, f: &mut std::fmt::Formatter) -> std::fmt::Result {
+        match self.time {
+            Some(v) => {
+                let now = Instant::now();
+                if v > now {
+                    let d = v.duration_since(now);
+                    write!(f, "timer={d:?} ")
```

Comment:
Was the idea here to use this as part of `Recovery`'s `Debug` impl? If so, I think the `timer=` prefix can be removed (below as well), so it can be used as `write!("timer={:?}", value)?;` in the other `Debug`.

Reply:
Yes, that was the idea.  The gcongestion implementation depends on Debug for LossDetectionTimer

It makes sense to use it also for congestion after the suggested cleanups.